python/lsst/daf/butler/registry/interfaces/_database.py

```diff
@@ -57,7 +58,9 @@
 from .._exceptions import ConflictingDefinitionError
 
 
-def _checkExistingTableDefinition(name: str, spec: ddl.TableSpec, inspection: List[Dict[str, Any]]) -> None:
+# TODO: method is called with list[ReflectedColumn] in SA 2, and
```

Comment:
So when we get SA2 we can make the type annotation explicit about that list contents?

Reply:
Right,  we should clean it up when we decide to drop 1.4 support.